helix-core/src/test/java/org/apache/helix/controller/stages/TestRebalancePipeline.java

```diff
@@ -45,10 +47,17 @@
 import org.apache.helix.model.Message;
 import org.apache.helix.model.Partition;
 import org.testng.Assert;
+import org.testng.annotations.AfterClass;
 import org.testng.annotations.Test;
 
 public class TestRebalancePipeline extends ZkUnitTestBase {
   private final String _className = getShortClassName();
+  private ExecutorService _executorService;
```

Comment:
Should we not have this variable in the scope of the class? Since this isn't being used throughout the class, we could just create the reference in the test method and shut down at the very end of the method. That way, we don't need to add the afterClass() method.